### DISCUSSION TITLE
Terminal Font Setting for LazyVim

### DISCUSSION THREAD

**Discussion by dean-boi** [coder/coder#14364] [Feature Request]
Hi, 

Is anyone using a Neovim distro in coder terminal? Is it possible to set fonts for the coder terminal web interface? For me LazyVim is displaying broken in the web terminal, but works properly in the code-server terminal.... 

![image](https://github.com/user-attachments/assets/76c2a509-2d6d-4bf4-9ec3-13eb61a82b2c)

![image](https://github.com/user-attachments/assets/992b7195-c314-456a-87aa-1c150ed29b88)

**Comment by matifali** [comment]
Hi, it is now possible to set fonts.
https://github.com/coder/coder/pull/17257
https://github.com/coder/coder/pull/17289

Would be available in release 2.22
